src/main/java/strategy/strategy/impl/BalaceReadActivity.java

```diff
@@ -0,0 +1,12 @@
+package strategy.strategy.impl;
+
+import dao.impl.AddToStorageImpl;
+import strategy.DoActivities;
+
+public class BalaceReadActivity implements DoActivities {
+
+    @Override
+    public void doActivity(String fruit, Integer number) {
+        new AddToStorageImpl().writeToStorage(fruit, number);
```

Comment:
Let's use one instance, but not create a new one each time this method is called